workspaces/apps-script-unit-test-agent/agent_context.yaml

```
@@ -0,0 +1,48 @@
+  - tests/apps-script.test.ts created
+  - All success scenarios tested
+  - All error scenarios tested
+  - 80% code coverage achieved
+  - Tests pass consistently
+
+test_scenarios:
+  success_cases:
+    - Valid script ID returns content
+    - Multiple files in script handled
+    - Different file types parsed correctly
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)